recipes/isl/build.sh

```diff
@@ -0,0 +1,4 @@
+./configure --prefix="$PREFIX" --with-gmp-prefix="$PREFIX"
```

Comment:
No, that's not the case.

We are specifying which system compiler to use. The situation with Mac is a little weird as they deprecated `gcc` some time ago. They have since moved everything over to `clang`. However, `gcc` has been left for backwards compatibility. That being said, it is actually a frontend to Clang/LLVM. So, compilation starts with `gcc`, but then proceeds through the LLVM framework. We still get linked to the standard GCC libraries.

However, this means by using `gcc`, we get pinned to a super old Mac OS 10.6, which is not even supported by Continuum AFAICT. Further this restricts us from getting nice features from `clang` in 10.7 like C++11 support. As much of our stack relies on C++11 support or at least C++0x support, we need to use `clang` on Mac to make sure we don't run into weird compatibility issues.

That being said, many programs try to pick up `gcc` first, which is what is happening here. All we are trying to do is to tell the configure script not to do that and use the supported system compiler (`clang`) instead of the unsupported one.